### DISCUSSION TITLE
milvus server fail to load config file

### DISCUSSION THREAD

**Discussion by spoinglee-sum** [milvus-io/milvus#3641] [Q&A and General discussion] [answered]
when I start milvus docker using follow script:
 docker run -d --name milvus_cpu_2 \
-p 19530:19530 \
-p 19121:19121 \
-p 9091:9091 \
-v /home/jcleethomas/lcl/milvus/db:/var/lib/milvus/db \
-v /home/jcleethomas/lcl/milvus/conf:/var/lib/milvus/conf \
-v /home/jcleethomas/lcl/milvus/logs:/var/lib/milvus/logs \
-v /home/jcleethomas/lcl/milvus/wal:/var/lib/milvus/wal \
milvusdb/milvus:0.7.0-cpu-d031120-de409b

I encounter error as below:

Milvus Release version: v0.7.0, built at 2020-03-11 09:15.58, with OpenBLAS library.
You are using Milvus CPU edition
Last commit id: de409be8d3c1e1127c88241a540f39b2b713c4cb
Loading configuration from: ../conf/server_config.yaml
Initializing log config from: ../conf/log_config.conf
S3 DISABLED !
Config check fail: Invalid file path
ERROR: Milvus server fail to load config file
Milvus server exit...

my server_config.yaml and log_config.conf is located in "  /home/jcleethomas/lcl/milvus/conf"
Should I transfer log_confg.conf to " /home/jcleethomas/lcl/milvus"

**Comment by shiyu22** [comment, marked as answer]
Please make sure your configuration file version corresponds to the Milvus version.

[The Milvus 0.7.0 configuration file](https://milvus.io/cn/docs/v0.7.0/cpu_milvus_docker.md#%E7%AC%AC%E4%B8%89%E6%AD%A5-%E4%B8%8B%E8%BD%BD%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6) is here.

And it is also highly recommended that you use the latest version of Milvus and the latest configuration file, you can [refer to it](https://milvus.io/cn/docs/v0.10.2/milvus_docker-cpu.md)!
